### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by x1p** [Ruinscraft/Panilla#17]
Random errors
```
[22:41:05 WARN]: java.lang.NullPointerException
[22:41:05 WARN]: at com.ruinscraft.panilla.bukkit.BukkitEnchantments.getMaxLevel(BukkitEnchantments.java:11)
[22:41:05 WARN]: at com.ruinscraft.panilla.api.nbt.checks.NbtCheck_ench.check(NbtCheck_ench.java:48)
[22:41:05 WARN]: at com.ruinscraft.panilla.api.nbt.checks.NbtChecks.checkAll(NbtChecks.java:98)
[22:41:05 WARN]: at com.ruinscraft.panilla.api.nbt.checks.NbtChecks.checkPacketPlayOut(NbtChecks.java:72)
[22:41:05 WARN]: at com.ruinscraft.panilla.craftbukkit.v1_13_R2.io.PacketInspector.checkPacketPlayOutSetSlot(PacketInspector.java:87)
[22:41:05 WARN]: at com.ruinscraft.panilla.api.io.IPacketInspector.checkPlayOut(IPacketInspector.java:33)
[22:41:05 WARN]: at com.ruinscraft.panilla.api.io.PlayerOutbound.write(PlayerOutbound.java:38)
[22:41:05 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:738)
[22:41:05 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:801)
[22:41:05 WARN]: at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:814)
[22:41:05 WARN]: at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:794)
[22:41:05 WARN]: at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:831)
[22:41:05 WARN]: at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1071)
[22:41:05 WARN]: at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:300)
[22:41:05 WARN]: at net.minecraft.server.v1_13_R2.NetworkManager.lambda$b$4(NetworkManager.java:214)
[22:41:05 WARN]: at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
[22:41:05 WARN]: at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
[22:41:05 WARN]: at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:313)
[22:41:05 WARN]: at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884)
[22:41:05 WARN]: at java.lang.Thread.run(Thread.java:748)
```
I'm not sure what causes it.
Panilla 1.3

**Comment by x1p** [comment]
Probably the same but longer```
[22:56:47 WARN]: java.lang.NullPointerException
[22:56:47 WARN]: at com.ruinscraft.panilla.bukkit.BukkitEnchantments.getMaxLevel(BukkitEnchantments.java:11)
[22:56:47 WARN]: at com.ruinscraft.panilla.api.nbt.checks.NbtCheck_ench.check(NbtCheck_ench.java:48)
[22:56:47 WARN]: at com.ruinscraft.panilla.api.nbt.checks.NbtChecks.checkAll(NbtChecks.java:98)
[22:56:47 WARN]: at com.ruinscraft.panilla.api.nbt.checks.NbtChecks.checkPacketPlayIn(NbtChecks.java:62)
[22:56:47 WARN]: at com.ruinscraft.panilla.craftbukkit.v1_13_R2.io.PacketInspector.checkPacketPlayInSetCreativeSlot(PacketInspector.java:64)
[22:56:47 WARN]: at com.ruinscraft.panilla.api.io.IPacketInspector.checkPlayIn(IPacketInspector.java:24)
[22:56:47 WARN]: at com.ruinscraft.panilla.api.io.PlayerInbound.channelRead(PlayerInbound.java:37)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[22:56:47 WARN]: at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:86)
[22:56:47 WARN]: at me.neznamy.tab.bukkit.packets.Packet$1.channelRead(Packet.java:59)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[22:56:47 WARN]: at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:86)
[22:56:47 WARN]: at de.robingrether.idisguise.management.channel.NettyChannelInjector$InjectedChannelHandler.channelRead(NettyChannelInjector.java:102)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[22:56:47 WARN]: at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:310)
[22:56:47 WARN]: at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:284)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[22:56:47 WARN]: at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:310)
[22:56:47 WARN]: at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:284)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[22:56:47 WARN]: at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[22:56:47 WARN]: at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1434)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[22:56:47 WARN]: at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[22:56:47 WARN]: at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:965)
[22:56:47 WARN]: at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:808)
[22:56:47 WARN]: at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:408)
[22:56:47 WARN]: at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:308)
[22:56:47 WARN]: at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884)
[22:56:47 WARN]: at java.lang.Thread.run(Thread.java:748)
```

**Comment by x1p** [comment]
Btw: I noticed that my compiled plugin differs in file size than the spigot one. In above example, I used the pre-compiled one from spigotmc.

**Comment by SrLeojaco** [comment]
Same issue here using 1.8

**Comment by dextonanderson** [comment]
I will fix this and have a 1.3.1 release by tomorrow

**Comment by dextonanderson** [comment]
Still not fixed
